modules/facilities_api/app/services/facilities_api/v1/lighthouse/middleware/errors.rb

```diff
@@ -28,5 +28,3 @@ def parse_body(env)
     end
   end
 end
-
```

Comment:
Is there a specific reason for removing the Faraday::Response.register_middleware and explicitly calling  FacilitiesApi::V1::Lighthouse::Middleware::Errors instead?

Reply:
Namespace collision, the lighthouse client was copied from the client in lib. It looks like it wasn't using the correct one even though they are currently identical

Reply:
I see you made a change here. Did something go wrong?